chart/templates/controller.yaml

```diff
@@ -16,13 +16,12 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: transport-controller-sa
-  namespace: {{.Values.wds_cp_name}}-system

```

Comment:
you removed the namespace from the service account, mistake?

Reply:
@francostellari that is very strange to see in one place that no namespace is specified while in the ClusterRoleBinding the namespace is specified (Release.Namespace). why not using Release.NS in both?

Reply:
from my understanding the result would be the same whether we specify `Release.Namespace` in serviceaccount or not (and then the helm chart namespace is used, which is exactly `Release.Namespace`). In general, I always prefer being explicit in such cases to keep consistency and being very clear.
I suggest doing the same for KS repo.